### PR TITLE
Add wurst

### PR DESCRIPTION
**Pull request by cmutel**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by cmutel** [comment]
@conda-forge/help-python, this is ready to review. Thanks for your time and support.

File: recipes/wurst/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ddb56c990945a73e42af3773c712e6ca144244d4584b144d0777a309ffe4e40d
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
```
Comment (marcelotrevisani) [inline recipes/wurst/meta.yaml:17]:
```suggestion
  host:
```

File: recipes/wurst/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - python
```
Comment (marcelotrevisani) [inline recipes/wurst/meta.yaml:18]:
```suggestion
    - python >=3.6
```

File: recipes/wurst/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    - country_converter
+    - docopt
+    - fiona
+    - numpy
+    - pandas
+    - python-json-logger
+    - shapely
+    - toolz
+    - tqdm
+    - wrapt
```
Comment (carterbox) [inline recipes/wurst/meta.yaml:36]: https://github.com/polca/wurst/blob/a30a6ebefb5584edb06ce2eff742b2e566e49ae0/setup.py#L17

These requirements do not match what the package claims.

File: recipes/wurst/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ddb56c990945a73e42af3773c712e6ca144244d4584b144d0777a309ffe4e40d
+
+build:
```
Comment (carterbox) [inline recipes/wurst/meta.yaml:12]: You need to lis[t entry points ](https://github.com/polca/wurst/blob/a30a6ebefb5584edb06ce2eff742b2e566e49ae0/setup.py#L28)in the conda-recipe. Please check the conda-forge docs section on noarch: python.

File: recipes/wurst/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    - shapely
+    - toolz
+    - tqdm
+    - wrapt
+
+test:
+  imports:
+    - {{ name }}
+  commands:
+    - pip check
```
Comment (carterbox) [inline recipes/wurst/meta.yaml:42]:
```suggestion
  commands:
    - pip check
    - wurst-cli --help
```
Test your CLI here.

**Review by carterbox — CHANGES_REQUESTED** [review]

**Comment by cmutel** [comment]
@m-rossi Will take over with a clean start. Thanks!

**Pull request closed without merging**